docs/hazmat/primitives/key-derivation-functions.rst

```diff
@@ -600,6 +600,154 @@ Different KDFs are suitable for different tasks such as:
         raises an exception if they do not match.
 
 
+.. currentmodule:: cryptography.hazmat.primitives.kdf.kbkdf
+
+.. class:: Mode
```

Comment:
I was going back and forth between the names. I ended up with just `Mode` because I couldn't find an example where an enum was prefixed. Happy to change it if there is a concern or ambiguity.